.github/workflows/ci.yml

```diff
@@ -35,7 +35,32 @@ jobs:
           slack_channel: ${{ secrets.SLACK_CHANNEL }}
           status: FAILED
 
+  filter-pr-changes:
+    runs-on: ubuntu-latest
```

Comment:
We're using `ubuntu-22.04` in other jobs, we could align this. The benefit of pinning is we're not surprised by upgrades, but we do have to update at some point.